docs/EN/Configuration/transition-qa.md

```diff
@@ -108,53 +108,101 @@ You must read extensively and adapt to a new approach to diabetes management.
+- <i>Some Trio users do a manual pre-bolus and skip entering carbs. Trio will predict a low BG until the meal kicks in and BG begins rising. If the pre-bolus is insufficient, Trio will give more insulin based on your settings. Yes, even without entering any carbohydrates. This approach will also result in a temporary high that is shorter than the completely unattended approach.</i>
+</details>
+<br>
+<details>
+  <summary><b>What do most people do?</b></summary>
+Most Trio users take a bit of all these approaches. They have an apple without entering anything into the app. Then, they pre-bolus for lunch because it is high in carbohydrates. Then, they use the calculator for dinner. This is all OK, and it comes down to the variation you accept for your BG.
+</details>
+<br>
 
-**Does everyone do it like that?**
-*No.* 
+## Coming From iAPS
 
-**What are the alternatives?**
+### iAPS Ver 2.3.3 or Earlier
 
-- *Some Trio users don't bolus for meals. They wait for Trio to detect rising BG and let Trio handle it. This usually leads to a temporary high BG, but Trio will get you back to target with the correct settings. If you're OK with a temporary high, then go ahead and try to skip bolus. You can start with small meals and tune your settings. You will need aggressive settings to let Trio give enough insulin.*
-- *Some Trio users do a manual pre-bolus and skip entering carbs. Trio will predict a low BG until the meal kicks in and BG begins rising. If the pre-bolus is insufficient, Trio will give more insulin based on settings. Yes, even without entering any carbs. This approach will also result in a temporary high that is shorter than the completely unattended approach.*
+Transitioning from iAPS 2.3.3 to Trio is fairly straight-forward currently. The screens and menus will be very familiar.
 
-**What do most people do?**
-*Most Trio users take a bit of all these approaches. They have an apple without entering anything into the app. Then, they pre-bolus for lunch because it was high carb. Then, they use the calculator for dinner. This is all OK, and it comes down to the variation you accept for your BG.*
+#### Settings
+
+Your settings will need to be entered manually, but all can be entered one-by-one from your iAPS settings without adjustment. If you have Nightscout, you can import your Pump Settings, Basal Profile, ISF, CR, Target Glucose, and previous CGM readings from the Nightscout Menu.
+<details>
+  <summary><b>Import Settings With Nightscout</b></summary>
+ 1. Connect Nightscout to your Trio app (If you've already connected your Nightscout site, skip to step 2)
+    * Open the 'Settings' Menu in the Trio app
```

Comment:
Nightscout settings can only be imported if the display units in Trio and Nightscout agree.
If they don't match, you will get an error message when you try to impor.t

Not sure how you want to phrase this - default for new app is mmol/L, so this only affects folks using mg/dL. Settings, Preferences, Glucose units.